From cd50df60787f98847a27fb223dc13d73613a17fa Mon Sep 17 00:00:00 2001
From: Jeremy Newton <alexjnewt@hotmail.com>
Date: Sat, 7 May 2022 00:03:48 -0400
Subject: [PATCH 2/2] Fix CL icd header include

Use CL public headers, not the the private header in the ICD loader source.

Fixes issues when building with ROCclr standalone package.
---
 src/fixme.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fixme.cpp b/src/fixme.cpp
index 3b6b55bc..9f5d4cc4 100644
--- a/src/fixme.cpp
+++ b/src/fixme.cpp
@@ -19,7 +19,7 @@
  THE SOFTWARE. */
 
 #include "vdi_common.hpp"
-#include <icd/loader/icd_dispatch.h>
+#include <CL/cl_icd.h>
 
 cl_icd_dispatch amd::ICDDispatchedObject::icdVendorDispatch_[] = {0};
 amd::PlatformIDS amd::PlatformID::Platform = {amd::ICDDispatchedObject::icdVendorDispatch_};
-- 
2.34.1

